tardis/io/logger/logger.py

```diff
@@ -136,31 +67,146 @@ def logging_state(log_level, tardis_config, specific_log_level):
                 logger.removeFilter(filter)
 
     if specific_log_level:
-        filter_log = FilterLog(LOGGING_LEVELS[logging_level])
+        filter_log = FilterLog([LOGGING_LEVELS[logging_level], logging.INFO, logging.DEBUG])
         for logger in tardis_loggers:
             logger.addFilter(filter_log)
     else:
         for filter in logger.filters:
             for logger in tardis_loggers:
                 logger.removeFilter(filter)
 
+log_outputs = {
+    "WARNING/ERROR": Output(layout=Layout(height='300px', overflow_y='auto')),
+    "INFO": Output(layout=Layout(height='300px', overflow_y='auto')),
+    "DEBUG": Output(layout=Layout(height='300px', overflow_y='auto')),
+    "ALL": Output(layout=Layout(height='300px', overflow_y='auto'))
+}
+
+tab = Tab(children=[log_outputs["WARNING/ERROR"], log_outputs["INFO"], log_outputs["DEBUG"], log_outputs["ALL"]])
+tab.set_title(0, "WARNING/ERROR")
+tab.set_title(1, "INFO")
+tab.set_title(2, "DEBUG")
+tab.set_title(3, "ALL")
+
+display(tab)
```

Comment:
do we really need to do it global level? shouldn't display be inside some function/class that gets called/instantiated?

If not, maybe move it together with other global level code at L184